modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DEWebContentField.testcase

```diff
@@ -203,7 +203,7 @@ definition {
 			webContentWebContent = "Web Content Title 2");
 	}
 
-	@description = "This is a stubbed description"
+	@description = "This is a test for LRQA-70033. This test verifies that is possible to link an Web Content"
```

Comment:
@guivilarim, what exactly do you mean with **link an Web Content**? Can you please clarify?